### PR TITLE
feat(apps/pano): show "Duzenle" and "Sil" on more options for the post

### LINKED ISSUE #634: Add a functionality to show "Duzenle" and "Sil" menu options if created post is belongs to session user

### PR DESCRIPTION
**Pull request by cansirin**
Closes #634 

This PR introduces a functionality to check if viewer and owner of the post are the same person, so we can show "Duzenle" and "Sil" options in the MoreOptions component.

### CODE REVIEW

File: apps/kampus/app/pano/PostListContainer.tsx [hunk 1]
```diff
@@ -16,17 +16,21 @@ const query = graphql`
...
+        ...PanoFeed_viewer
+      }
     }
   }
 `;
 
 export const PostListContainer = (props: Props) => {
   const queryRef = useSerializablePreloadedQuery(props.preloadedQuery);
   const data = usePreloadedQuery(query, queryRef);
+  console.log(data);
```
Comment (fdemir) [inline apps/kampus/app/pano/PostListContainer.tsx:29]: remove

**Review by Ketcap — APPROVED** [review]

File: apps/kampus/app/pano/features/post-list/MoreOptions.tsx [hunk 2]
```diff
@@ -20,91 +23,120 @@ import {
...
+        }
+      }
+    `,
+    key
+  );
 
-export const MoreOptionsDropdown = ({ post, shareUrl }: Props) => {
-  const user = { username: "John Doe" };
-  console.log(post, shareUrl);
+const remotePostMutation = graphql`
```
Comment (usirin) [inline apps/kampus/app/pano/features/post-list/MoreOptions.tsx]:
```suggestion
const removePostMutation = graphql`
```

File: apps/kampus/app/pano/PanoFeed.tsx [hunk 3]
```diff
@@ -17,9 +18,11 @@ const fragment = graphql`
   @refetchable(queryName: "PanoFeedPaginationQuery") {
     panoFeed(first: $first, after: $after, last: $last, before: $before)
       @connection(key: "PanoFeedFragment__panoFeed") {
+      __id
       edges {
         cursor
         node {
+          __id
```
Comment (usirin) [inline apps/kampus/app/pano/PanoFeed.tsx]:
```suggestion
```

File: apps/kampus/app/pano/features/post-list/PostItem.tsx [hunk 4]
```diff
@@ -34,31 +34,49 @@ const usePanoPostFragment = (key: PostItem_post$key | null) =>
...
 export const PostItem = (props: PostItemProps) => {
   const post = usePanoPostFragment(props.post);
+  const viewer = usePanoViewerFragment(props.viewerRef);
 
   if (!post) {
     return null;
   }
 
+  console.log(viewer, "viewer");
+
```
Comment (usirin) [inline apps/kampus/app/pano/features/post-list/PostItem.tsx]:
```suggestion
```

**Review by usirin — APPROVED** [review]

**Pull request merged**